test/module/shared_model/cryptography/crypto_usage_test.cpp

```diff
@@ -185,28 +209,46 @@ TEST_F(CryptoUsageTest, SameQueryHashAfterSign) {
  * @when verify transaction
  * @then transaction is not verified
  */
-TEST_F(CryptoUsageTest, UnsignedTransaction) {
-  ASSERT_THAT(verify(*transaction), kNoSignatureMatcher);
+TYPED_TEST(CryptoUsageTest, UnsignedTransaction) {
+  ASSERT_THAT(this->verify(*this->transaction), kNoSignatureMatcher);
 }
 
 /**
  * @given properly signed transaction
  * @when verify transaction
  * @then transaction is verified
  */
-TEST_F(CryptoUsageTest, SignAndVerifyTransaction) {
-  signer.sign(*transaction);
+TYPED_TEST(CryptoUsageTest, SignAndVerifyTransaction) {
+  this->signer.sign(*this->transaction);
 
-  EXPECT_EQ(verify(*transaction), std::nullopt);
+  EXPECT_EQ(this->verify(*this->transaction), std::nullopt);
 }
 
 /**
  * @given transaction with incorrect sign
  * @when verify transaction
  * @then transaction is not verified
  */
-TEST_F(CryptoUsageTest, SignAndVerifyTransactionkWithWrongSignature) {
-  signIncorrect(*transaction);
+TYPED_TEST(CryptoUsageTest, SignAndVerifyTransactionkWithWrongSignature) {
+  this->signIncorrect(*this->transaction);
 
-  EXPECT_THAT(verify(*transaction), kBadSignatureMatcher);
+  EXPECT_THAT(this->verify(*this->transaction), kBadSignatureMatcher);
+}
+
+/**
+ * @given Initialized keypiar with _concrete_ algorithm
+ * @when sign date without knowledge of cryptography algorithm
+ * @then check that siganture valid without clarification of algorithm
```

Comment:
```suggestion
 * @given Initialized keypair with _concrete_ algorithm
 * @when sign data without knowledge of cryptography algorithm
 * @then check that signature is valid without clarification of algorithm
```
However, the description does not fit the test, please fix.